lib/ReactViews/Guide/satellite-guidance.js

```diff
@@ -0,0 +1,38 @@
+export const guideData = [
+  {
+    imageSrc: require("../../../wwwroot/images/guides/satellite-feature.jpg"),
+    title: "Satellite Imagery: How do I…?",
+    body:
+      "A super-quick guide to getting the most out of historical satellite imagery.",
+    prevText: "No thanks",
+    nextText: "View guide"
+  },
+  {
+    imageSrc: require("../../../wwwroot/images/guides/satellite-zoom.jpg"),
+    title: "Where is my satellite imagery?",
+    body:
+      "You may need to adjust your zoom level to locate near real-time satellite imagery on the map.",
```

Comment:
Suggest rewording from 
"You may need to adjust your zoom level to locate near real-time satellite imagery on the map.",
to 
"You may need to adjust your zoom level to make satellite imagery visible on the map.",

Rationale being
- it's not just "near real-time" imagery effected
- and zooming in doesn't "locate" the imagery

Reply:
Thanks that's a better, more generic copy that applies!